src/components/Person/Person.jsx

```diff
@@ -1 +1,11 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age !== undefined && <p className="Person__age">{`I am ${person.age}`}</p>}
+    <p className="Person__partner">
+      {!person.isMarried ? 'I am not married' : `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}
```

Comment:
Consider splitting this logic a bit and moving it from your jsx. Avoid using nested ternary